exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Column.kt

```diff
@@ -67,7 +67,7 @@ class Column<T>(val table: Table, val name: String, override val columnType: ICo
 
         when {
             !isPKColumn && isSQLiteAutoIncColumn -> tr.throwUnsupportedException("Auto-increment could be applied only to primary key column")
-            isSQLiteAutoIncColumn && !isOneColumnPK() && table.primaryKey != null -> append(currentDialect.dataTypeProvider.integerType())
+            isSQLiteAutoIncColumn && (!isOneColumnPK() || table.isCustomPKNameDefined()) && table.primaryKey != null -> append(currentDialect.dataTypeProvider.integerType())
             else -> append(colType.sqlType())
```

Comment:
When using SQLite :  If `isOneColumnPK()` = true and the column type is `integerAutoincType`, then `colType.sqlType()` returns : `"INTEGER PRIMARY KEY AUTOINCREMENT"` at the same time when defining a costom primaryKey name, a primary key constraint is generated too. That's why we get the error.
Here I added a condition to prevent defining primary key multiple times.